content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -238,8 +238,8 @@ different Kubernetes components.
 | `CSIPersistentVolume` | `true` | Beta | 1.10 | 1.12 |
 | `CSIPersistentVolume` | `true` | GA | 1.13 | - |
 | `CSIServiceAccountToken` | `false` | Alpha | 1.20 | 1.20 |
-| `CSIServiceAccountToken` | `true` | Beta | 1.21 | 1.21 |
-| `CSIServiceAccountToken` | `true` | GA | 1.22 | |
+| `CSIServiceAccountToken` | `true` | Beta | 1.21 | 1.22 |
```

Comment:
This is wrong. The gate becomes GA in 1.22. So that last column should be `1.21`.

Reply:
done